### PR TITLE
Improve the pull-request subcription notification format by adding the description and files statistics

### PR DESCRIPTION
**Pull request by joker-eph**

### CODE REVIEW

**Comment by joker-eph** [comment]
See https://github.com/llvm/llvm-project/pull/65439#issuecomment-1712368466 for a test

File: llvm/utils/git/github-automation.py [hunk 1]
```diff
@@ -199,7 +246,6 @@ def extract_commit_hash(arg: str):
 
 
 class ReleaseWorkflow:
-
     CHERRY_PICK_FAILED_LABEL = "release:cherry-pick-failed"
```
Comment (tstellar) [inline llvm/utils/git/github-automation.py:249]: Stray whitespace change?

**Review by tstellar — APPROVED** [review]
LGTM.  Thanks for doing this, I like the format.

File: llvm/utils/git/github-automation.py (hunk 1 above)
Comment (joker-eph) [reply llvm/utils/git/github-automation.py:249]: This is black formatting!

File: llvm/utils/git/github-automation.py [hunk 2]
```diff
@@ -77,23 +85,62 @@ def __init__(self, token: str, repo: str, pr_number: int, label_name: str):
...
+                print(f"(from {file.previous_filename})")
+            diff_stats += "\n"
+        diff_stats += "\n"
+
+        # Get the diff
+        try:
+            patch = requests.get(self.pr.diff_url).text
+        except:
+            patch = ""
+        diff_stats += "\n<pre>\n" + patch
```
Comment (jayfoad) [inline llvm/utils/git/github-automation.py:113]: Hi, I'm seeing this `<pre>` getting escaped to `&lt;pre&gt;`, so it no longer has the desired effect and the diff looks horrible in the github web UI. Example: https://github.com/llvm/llvm-project/pull/66188#issuecomment-1717259869

Comment (joker-eph) [reply llvm/utils/git/github-automation.py:113]: This was changed in #66037 , I'm not totally sure why

Comment (joker-eph) [reply llvm/utils/git/github-automation.py:113]: Fixing in https://github.com/llvm/llvm-project/pull/66322

**Pull request merged**